sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAccessControlClientBuilder.java

```diff
@@ -309,6 +351,7 @@ public KeyVaultAccessControlClientBuilder pipeline(HttpPipeline pipeline) {
      *
      * @return The updated {@link KeyVaultAccessControlClientBuilder} object.
      */
+    @Override
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores the value set in this method. We should probably add a note like you did in other methods on this same class.